rpm/ngfd.spec

```diff
@@ -25,6 +26,8 @@ BuildRequires:  pkgconfig(mce)
 BuildRequires:  pkgconfig(profile)
 BuildRequires:  pkgconfig(libcanberra)
 BuildRequires:  doxygen
+Obsoletes:      tone-generator <= 1.5.4
+Provides:       tone-generator > 1.5.4
```

Comment:
Shouldn't require such thing as it will anyway remove those while this package is installed.